public/index.html

```diff
@@ -262,11 +262,51 @@
 <!-- SIDEBAR CONTENT -->
 <!-- See http://plugins.adchsm.me/slidebars/ for additional info -->
 <div class="sb-style-push sb-slidebar sb-right">
-    Yo.
+    <nav class="navbar navbar-inverse" role="navigation">
+        <div class="sidebar-container">
+                <ul class="nav nav-stacked">
+                    <li>
+                        <a href="#plate-filter" data-toggle="collapse">Plate</a>
+                        <div class="collapse container-fluid filter-content" id="plate-filter">
+                            <div class="row">
+                                <div class="col-md-6 filter-items" id="plate-1"></div>
+                                <div class="col-md-6 filter-items" id="plate-2"></div>
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <a href="#row-filter" data-toggle="collapse">Row</a>
+                        <div class="collapse container filter-content" id="row-filter">
+                            <div class="row">
+                                <div class="col-md-6 filter-items" id="row-1"></div>
+                                <div class="col-md-6 filter-items" id="row-2"></div>
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <a href="#col-filter" data-toggle="collapse">Column</a>
+                        <div class="collapse container filter-content" id="col-filter">
+                            <div class="row">
+                                <div class="col-md-4 filter-items" id="col-1"></div>
+                                <div class="col-md-4 filter-items" id="col-2"></div>
+                                <div class="col-md-4 filter-items" id="col-3"></div>
+                            </div>
+                        </div>
+                    </li>
+                    <li>
+                        <a href="#gene-filter" data-toggle="collapse">Gene</a>
```

Comment:
Let's move this to the top of the filter list. Am I supposed to be able to click on this box and type? Cos it's not working for me...